subprojects/core/src/integTest/groovy/org/gradle/api/InitScriptExecutionIntegrationTest.groovy

```diff
@@ -271,6 +271,15 @@ initscript {
         output.contains("Project 'root'")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/17555")
+    def "init script file is a dotfile"() {
+        def initScript = file('.empty')
+        initScript << 'println "greetings from user home"'
```

Comment:
It would be better if we replace this to something more meaningful like "greetings from empty init script". I assume this was copied from somewhere else.

Reply:
Can't deny that...